Commonalities/documentation/Working/General-principles-doc.md

```diff
@@ -43,6 +44,11 @@
  <li> Versioning </li>
  <ul>    
     <li> Strategy </li>
+    <ul>
+         <li> Agreement on major version  </li>
```

Comment:
I think there should be another bullet (after the first): 'Naming convention for versions' . Then the actual choice of naming convention (e.g. semantic versioning 2.0, as suggested by Bart) could be included within the bullet content.